### ISSUE TITLE
Portmaster says it's being Interfered by VPN when VPN is not being used

### ISSUE THREAD

**Issue by Rexadev** [safing/portmaster#908]
![image](https://user-images.githubusercontent.com/62152714/193291081-da84694b-063d-4b6a-b078-27665f3f5368.png)
![image](https://user-images.githubusercontent.com/62152714/193291215-10aa446b-0d49-44ab-8be6-7c319799b64b.png)
![image](https://user-images.githubusercontent.com/62152714/193291130-791147ee-5859-4890-81c8-58e5ceb5ebd6.png)

**Pre-Submit Checklist**:

- Check applicable sources for existing issues:
  - [Windows Known Issues](https://docs.safing.io/portmaster/install/windows#known-issues)
  - [Linux Known Issues](https://docs.safing.io/portmaster/install/linux#compatibility)
  - [Github Issues](https://github.com/safing/portmaster/issues?q=is%3Aissue+label%3Abug)

**What happened**:

**What did you expect to happen?**:

**How did you reproduce it?**:

**Debug Information**:

**Comment by uzuki-P** [comment]
Are you on Windows?
Is there any DNS configuration on the network setting?
![image](https://user-images.githubusercontent.com/16524713/193401463-520047c0-e7a9-4ec7-8fde-af671c980782.png)

Try to use DHCP it might fix the problem

**Comment by dhaavi** [comment]
This can be because of many things.
Please post debug info.

**Comment by WanderLanz** [comment]
Just got this same behavior today, there's a flag on VirusTotal, maybe AV is mucking about?

**Comment by WanderLanz** [comment]
fixed itself before I got around to debugging, oh well, Microsoft things I guess

**Comment by numb699** [comment]
I get this as well now. No VPN installed or in use. Fully updated Windows 11 standard Defender setup. 
![bilde](https://user-images.githubusercontent.com/58867800/197484572-c7a92ba6-dc00-41d1-83e2-58c6337d1275.png)
[debug.txt](https://github.com/safing/portmaster/files/9850162/debug.txt)
The notification disappears after a few minutes and the Secure sign is back.

**Comment by dhaavi** [comment]
Were you changing settings when this happened?

**Comment by numb699** [comment]
> Were you changing settings when this happened?

No, this seem to happen at random intervals. Happens with the app showing/open and sometimes the light turns red and shows this when you open the app.

**Comment by R-Studio** [comment]
I have the same issue and I think it is related to Windows Defender, but I have no idea to how to fix this. 😔

**Comment by WanderLanz** [comment]
I had just happened to be looking through AutoRuns to re-enable an OEM driver when I happened upon an Unverified driver for Portmaster:

"C:\ProgramData\Safing\Portmaster\updates\windows_amd64\kext\portmaster-kext_v1-0-14.sys", 

Is there perhaps a problem with getting verified driver updates causing defender to treat Portmaster activity as untrusted when it prevents some telemetry (although VirusTotal no longer flags portmaster)?

There is v1-0-15 and v1-0-16 but they are not in AutoRuns. This seems like a Windows engineer moment.
